templates/default/env/configs.erb

```diff
@@ -14,3 +14,6 @@ SECRET_SIDECAR_IMAGE=<%= @secret_sidecar_image %>
 SECRET_STORAGE_BACKEND=<%= @secret_storage_backend || 'SecretStorage::DbBackend' %>
 SECRET_TOKEN=<%= @secret_token %>
 MEMCACHE_SERVERS=<%= Array(@memcache_servers).join(',') %>
+<% @custom_config_items.each do |conf_name, conf_value| %>
```

Comment:
Is there a reason you chose to nest custom_config_items rather than have it higher level ?